tests/test_data/exodus-config.json

```diff
@@ -2,159 +2,197 @@
     "origin_alias": [
         {
             "src": "/content/origin",
-            "dest": "/origin"
+            "dest": "/origin",
+            "exclude_paths": []
```

Comment:
I suppose it's OK to leave a couple of aliases with `"exclude_paths": []`, but to accurately reflect the current state of the CDN, please omit the exclude_paths key/pair altogether instead of specifying an empty list (`"exclude_paths": []`), so that we can test that the CDN can handle alias resolutions when an alias does not contain an `exclude_paths` key (i.e., the current state of **all** aliases on the CDN).